### ISSUE TITLE
About the datasets

### ISSUE THREAD

**Issue by ucasiggcas** [ifnspaml/perceptual-weighting-filter-loss#2]
Hi,
Glad to see the source codes.
Now I find the signals in the [link](http://spandh.dcs.shef.ac.uk/gridcorpus/#downloads)
Should I download the first two columns ?
And then put them at the AudioData/ ?

thx

**Comment by ucasiggcas** [comment]
And I can't find the CHiME-3 noises data from the [paper](https://github.com/ifnspaml/Perceptual-Weighting-Filter-Loss/files/3444877/noise_barker2015.pdf)
Could U please help me ?

Thanks a lot

**Comment by ucasiggcas** [comment]
Could you please help me to get the CHiME-3 noises data?
Or where can I find the data ?
If you can supply the link will be good.
Thanks.

**Comment by ucasiggcas** [comment]
eh,
I find the CHiME-3 [here](https://catalog.ldc.upenn.edu/LDC2017S24)
if you have money,you can download 
or else give up
SO I give up
and I will use my own noise data

**Comment by ZiyueZhao** [comment]
Hi,

Thanks for your interests in the work!

The clean speech signals are from Grid corpus, as the link you find. Therein, the first column contains all the audio signals at 25 kHz sampling rate. As 16 kHz sampling rate is used in the paper, a downsampling processing is needed before you put all signals to the correlated directories. You may either use ITU-T softwares, or other tools for downsampling. Please note that, in order to keep a balance of male/female speakers in the training/validation and test dataset, the speaker indices are deliberately selected, and may differ from the original speaker indices. It can be find in GitHubTrain_part_1_CleanAndNoisyMixture.m (cf. comments in the section of "Input directories").

The ChiMe-3 noise data can be found here: https://catalog.ldc.upenn.edu/LDC2017S24. As licences are needed, so I'm sorry that I cannot distribute this dataset.

Best

**Comment by ZiyueZhao** [comment]
Yes, you may use other noise datasets (maybe open source or your own noise datasets) for your training and testing. Some modifications in the preparation of the mixture are therefore needed, however, other parts of the source code can still be used in this case.
